nerfstudio/scripts/render.py

```diff
@@ -313,6 +313,14 @@ class RenderTrajectory:
     """How to save output data."""
     interpolation_steps: int = 10
     """Number of interpolation steps between eval dataset cameras."""
+    order_poses: bool = False
+    """Whether to order camera poses by proximity"""
+    adjust_frame_rate: bool = False
+    """Whether to adjust the frame rate of the output video."""
```

Comment:
Note that this is only for interpolated paths

Reply:
Done